readme.md

```diff
@@ -9,6 +9,25 @@ Shadow is a new novel web engine made (almost) entirely in JS from scratch.
 - [Introducing Shadow](https://goose.icu/introducing-shadow/)
 - [Shadow Devlog #1](https://goose.icu/shadow-devlog-1/)
 
+## Contributing
+
+### Installation
+
+```sh
```

Comment:
This isn't actually needed, just for the weird mostly unusable GTK Node frontend. This installation step can be taken out and the section could just be starting a local HTTP server in the repo dir.

Reply:
Got it! Have updated accordingly and force-pushed over the previous commit.